README.md

```diff
@@ -1,21 +1,21 @@
-# PlandoRandomSettings
-This script will generate a plando json file that can be fed into the Ocarina of Time randomzier with random values for each of the randomizer settings.
-This script allows us to blindly randomize anything and (nearly) everything in the ranomizer, rather than just the natively supported random settings.
+# Random Settings Generator
+This script will generate a patch file for The Legend of Zelda Ocarina of Time Randomizer with randomly selected randomizer settings.
+This script allows its user to randomize every setting in the Randomizer, not just settings that are natively supported to be randomized.
 
 ## Instructions
-1. Download the dev-R branch of Ocarina of Time Randomizer. Version v5.2.65 R-11 or newer  (https://github.com/Roman971/OoT-Randomizer).
-2. Navigate to the folder that contains `Gui.py` and make a new folder named `plando-random-settings`.
-3. Place the contents of this repository in this new folder.
-4. Run `PlandoRandomSettings.py` by either double clicking via the command line `$ python3 PlandoRandomSettings.py` to generate `blind_random_settings.json`.
-5. Use this file as a plando file when you generate your randomizer seed.
+1. Have Python 3.7 (or newer) installed. This is also a requirement of the randomizer.
+2. Download the zip file of the source code from the release page and unzip anywhere: https://github.com/matthewkirby/plando-random-settings/releases
+3. Run the code by double clicking `RandomSettingsGenerator.py` or running `python3 RandomSettingsGenerator.py` via the command line.
```

Comment:
The Windows executable will just be python.exe. So if you check the installation option "Add Python to environment variables" in the Windows installer, you would use `python RandomSettingsGenerator.py`.
`